<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeStaticSettingsDialog</class>
 <widget class="QDialog" name="ModeStaticSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Czerwony</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSlider" name="sRed">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="bSave">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="sBlue">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Niebieski</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Zielony</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="sGreen">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="bExit">
     <property name="text">
      <string>Wyjd≈∫</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" rowspan="2" colspan="3">
    <widget class="QWidget" name="wDisplayColor" native="true">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
